src/components/application_manager/test/commands/mobile/perform_audio_pass_thru_test.cc

```diff
@@ -108,104 +125,503 @@ class PerformAudioPassThruRequestTest
     Mock::VerifyAndClearExpectations(&mock_message_helper_);
   }
 
-  void ResultCommandExpectations(MessageSharedPtr msg,
-                                 const std::string& info) {
+  void ResultCommandInfoExpectations(MessageSharedPtr msg,
```

Comment:
@tkireva You have separate function to check command code. Why do you duplicate this functionality?